examples_tests/61.BlitFilterTest/alpha_test/descriptors.glsl

```diff
@@ -0,0 +1,37 @@
+#ifndef _NBL_GLSL_BLIT_ALPHA_TEST_DESCRIPTORS_INCLUDED_
```

Comment:
tbh, alpha test could use the same descriptor set as blit, as long as the atomic pass counter + histogram SSBOs are unified

Reply:
Resolved.